src/components/controls/view/km-lstring-value.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div :id="$attrs.id" class="scbd-controls km-form-group form-group">
```

Comment:
what are these css classes `km-formp-group form-group`
why is there no km-lstring-value